wscl-issues/draft/sequence-definition

```diff
@@ -0,0 +1,69 @@
+Issue:          SEQUENCE-DEFINITION
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   16-May-24, Version 1 by Jin-Cheng Guu
+References:     SEQUENCE
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the description of the system
+  class SEQUENCE [a] conflicts with the description of the Sequence Concepts
+  [b], where
+
+    [a]: The types vector and the type list are disjoint subtypes of type
+    sequence, but are not necessarily an exhaustive partition of sequence.
+
+    [b]: A sequence is an ordered collection of elements, implemented as
+    either a vector or a list.
+
+Proposal: (SEQUENCE:A-SEQUENCE-CAN-BE-SOMETHING-ELSE)
+
+  This proposal changes the description of the Sequence Concepts so that [b]
+  instead reads:
+
+    > A sequence is an ordered collection of elements, implemented either as a
+    vector, a list, or possibly some other implementation-defined object.
+
+Test Cases:
+
+  N/A (Test cases are difficult to provide, because conformating
+  implementations are not expected to expose its definition of a SEQUENCE
+  programmatically.)
+
+Rationale:
+
+  This is a plain contradiction in dpANS.
```

Comment:
You have already stated that it is a contradiction in the Problem Description. The Rationale section should be justification for your solution suggested in the Proposal section. 

I suggest:

For Common Lisp implementations that support user defined sequences via a mechanism like Extensible Sequences LIST and VECTOR will not be an exhaustive partition of SEQUENCE. In this case [a] is an accurate statement, but [b] will not be sufficient.